tvx/exec_lotus.go

```diff
@@ -29,38 +30,66 @@ var execLotusCmd = &cli.Command{
 		&cli.StringFlag{
 			Name:        "file",
 			Usage:       "input file",
-			Required:    true,
 			Destination: &execLotusFlags.file,
 		},
+		&cli.BoolFlag{
+			Name:        "stdin",
+			Usage:       "",
+			Destination: &execLotusFlags.stdin,
+		},
```

Comment:
can't this be implicit when file is not set?

Reply:
i agree, that's simpler. fixed.